src/Elmish.WPF/DynamicViewModel.fs

```diff
@@ -8,6 +8,23 @@ open Microsoft.Extensions.Logging
 
 open BindingVmHelpers
 
+/// Represents all necessary data used to create a binding.
+type Binding<'model, 'msg> =
```

Comment:
Why move this here instead of to the `Binding.fs` file?

Reply:
It's used in this file.